tests/DependencyInjection/HWIOAuthExtensionTest.php

```diff
@@ -38,15 +38,23 @@ protected function tearDown(): void
         unset($this->containerBuilder);
     }
 
-    public function testConfigurationThrowsExceptionUnlessFirewallNameSet(): void
+    public function testConfigurationTriggersDeprecatedNoticeIfFirewallNamesSet(): void
     {
-        $this->expectException(InvalidConfigurationException::class);
+        $errors = [];
+        // @phpstan-ignore-next-line Handler parameters.
+        set_error_handler(function (int $errno, string $error) use (&$errors) {
+            $errors[] = $error;
+        }, \E_USER_DEPRECATED);
```

Comment:
I would go with existing tools:
- https://symfony.com/doc/current/components/phpunit_bridge.html#write-assertions-about-deprecations
- https://github.com/symfony/deprecation-contracts

Reply:
thx for hint. is done